src/Microsoft.AspNetCore.RuntimeStore/Microsoft.AspNetCore.RuntimeStore.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\Microsoft.AspNetCore.All\Microsoft.AspNetCore.All.csproj " />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.AzureAppServices.HostingStartup" Version="$(AspNetCoreVersion)" PrivateAssets="None" />
+    <PackageReference Include="Microsoft.AspNetCore.AzureAppServicesIntegration" Version="$(AspNetCoreVersion)" PrivateAssets="None" />
+    <PackageReference Include="Microsoft.AspNetCore.ApplicationInsights.HostingStartup" Version="$(AspNetCoreVersion)" PrivateAssets="None" />
+    <PackageReference Include="Microsoft.Extensions.Logging.AzureAppServices" Version="$(AspNetCoreVersion)" PrivateAssets="None" />
+  </ItemGroup>
+
+  <Target Name="CollectDeps" DependsOnTargets="Restore;RunResolvePackageDependencies">
+      <ItemGroup>
+        <DepsFiles Include="%(FileDefinitions.ResolvedPath)" Condition="$([System.Text.RegularExpressions.Regex]::IsMatch('%(Identity)', '.*?\.HostingStartup\.deps\.json')) "/>
+      </ItemGroup>
+      
+      <Copy SourceFiles="%(DepsFiles.Identity)" DestinationFolder="$(DepsOutputPath)\Microsoft.NETCore.App\2.0.0\" />
```

Comment:
<Copy SourceFiles="@(DepsFiles)" ...

You don't need to copy each file over individually.